src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/XmlnsCache.cs

```diff
@@ -253,6 +254,15 @@ private void GetNamespacesFromDefinitionAttr(CustomAttributeData data, out strin
                 else
                     throw new ArgumentException(SR.Get(SRID.ParserAttributeArgsHigh, "XmlnsDefinitionAttribute"));
             }
+            IList<CustomAttributeNamedArgument> namedArguments = data.NamedArguments;
+            for (int i = 0; i < namedArguments.Count; i++)
+            {
+                var namedArgument = namedArguments[i];
+                if (namedArgument.MemberName == "AssemblyName")
```

Comment:
Yes... I almost did that too, but was trying to follow the pattern of current code - I don't really see the use of nameof much in the codebase, if at all.

Reply:
Most of the codebase predates nameof, that’s all. It’s not a deliberate convention to avoid use of it etc.. Newer code does use it. 

Use of nameof should be fine except where translatable strings are involved (they typically appear as SRID.Name).

Reply:
There's another issue here though: You can only use nameof(...) for types that are referenced, but the `XmlnsDefinitionAttribute` type isn't referenced in this project.